### ISSUE TITLE
Good one!

### ISSUE THREAD

**Issue by stychu** [giuseppelt/httpc#3]
Just found this mentioned on some reddit post.
I've been doing some research for the past few months around functional, RPC like framework (correlated with tRPC, ts-rest, zodios etc.) but haven found anything fulfilling my needs. But this seems like the one! I love the idea and simplicity. This is what I need.

What is current state of the project?

I would love to contribute to the project to help growth and fast v1 release <3 (currently busy at work so it can be hard atm).
For sure you have my star and follow. Looking forward to what this can emerge into.
Keep good work :)

**Comment by giuseppelt** [comment]
Hi! Thanks for your words!

I could have answered earlier, but I let some days pass to better formulate my thoughts.

The httpc development is active. I use it in every project I do. I'm using it with dozen production servers and webapp clients.

But the status for others collaborators is not good:
- the v0.2 branch has lot of refactoring, breaking changes and improvements. But there's no examples.
- the current documentation is way behind
- it's still very experimental, with lot of missing features, rough edges
- there're core ideas inside that are good, but some outer layer are just patched up, and they require lot of rework
- naming is a total mess
- the API is not stable at all. As I develop httpc in sync with all my projects, I change the API quite frequently. For me is not a problem, because all my repos are synched together, so changes propagate nicely. But it's not good for a facing library.

When I started the httpc project, my aim was to dedicated to it a good chuck of my time. But I failed to make it more popular, and I don't have any connection.

So now basically I develop it within my projects and add missing features or refinements just to make my "main" projects run.

If you still want to give it a try, maybe we can follow/build together a use case or an example.
In other words, I can tell you how to setup a project and you go on from there, asking if you need features or you can't find out how to do something.

Let me know what you think.

**Comment by giuseppelt** [comment]
PS, if you active on twitter, follow me https://x.com/giuseppelt 
Sometime I post httpc related stuff there

**Comment by mooreorless555** [comment]
I really like this approach! I really hope it gets more popular.
